projects/erigon_focil.md

```diff
@@ -0,0 +1,94 @@
+# Fork-Choice Enforced Inclusion Lists (FOCIL) For Erigon 
+
+## Motivation
+
+[EIP-7805](https://eips.ethereum.org/EIPS/eip-7805) is a potential candidate for the glamsterdame upgrade which focuses on improving the block building on Ethereum. Currently block building is centralized and not resistant to censorship. With this EIP ethereum expects to fix these issues and improve the block building capabilities of the chain.
+
+FOCIL enforces some rules when the blocks are being built. It forces the block to include some transactions via IL's(inclusion lists). These inclusion lists are built by random committee members. These inclusion lists can be created using the ruling chosen by the client software. Once these lists are created their inclusion in the block are checked for the block to be accepted/rejected.
+
+Block building is very centralized today and FOCIL can help mitigate it.
```

Comment:
It doesn't help to decentralize the block building, only enables validators to enforce rules on these centralized actors.